src/tokenized-line.coffee

```diff
@@ -59,7 +74,7 @@ class TokenizedLine
     screenColumn = 0
     currentBufferColumn = 0
     for token in @tokens
-      break if currentBufferColumn > bufferColumn
+      break if currentBufferColumn + token.bufferDelta > bufferColumn
```

Comment:
Never mind. After completing specs around this I think we can just use your version, as it's symmetrical with `bufferColumnForScreenColumn` :+1: